README.md

```diff
@@ -1,13 +1,13 @@
 # Akka Cluster Management
 
-[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/akka/akka?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/akka/akka)
 
 This repository contains interfaces to interact with an Akka Cluster.
 
 Documentation
 -------------
 
-See [reference](http://developer.lightbend.com/docs/akka-cluster-management/0.1-RC1/) and [API](http://developer.lightbend.com/docs/api/akka-cluster-management/0.1-RC1/akka/cluster/http/management/index.html) documentation pages.
+See [reference](http://developer.lightbend.com/docs/akka-cluster-management/latest/) and [API](http://developer.lightbend.com/docs/api/akka-cluster-management/latest/akka/cluster/http/management/index.html) documentation pages.
```

Comment:
OK, I confused latest and current to be the same. I will remove this change and fix the conflict